src/config.rs

```diff
@@ -109,4 +109,7 @@ pub struct Config {
 
     /// The maximum number of search results
     pub max_results: Option<usize>,
+
+    /// Whether to prefix each relative path with "./"
+    pub no_strip: bool,
```

Comment:
There is a mental overhead in all of the code where I need to negate everything in my head, because this is called "no strip". How about calling this: `strip_cwd_prefix` and reversing all the logic in the code.